utils/_weblog.py

```diff
@@ -221,29 +228,49 @@ def request(
         status_code = None
         response_headers: CaseInsensitiveDict = CaseInsensitiveDict()
         text = None
-
-        try:
-            req = requests.Request(
-                method, url, params=params, data=data, json=json, files=files, headers=headers, cookies=cookies
-            )
-            r = req.prepare()
-            r.url = url
-            logger.debug(f"Sending request {rid}: {method} {url}")
-
-            s = requests.Session() if self._session is None else self._session
-            response = s.send(r, timeout=timeout, stream=stream, allow_redirects=allow_redirects)
-            status_code = response.status_code
-            response_headers = response.headers
-            text = response.text
-
-        except Exception as e:
-            logger.error(f"Request {rid} raise an error: {e}")
-        else:
-            logger.debug(f"Request {rid}: {response.status_code}")
-            if response.status_code == HTTPStatus.NOT_FOUND:
-                logger.error(
-                    "💡 if your test is failing, you may need to add missing_feature for this weblog in manifest file."
+        retries = 1
```

Comment:
Could you add a small comment explaining why we do this retry specifically for uwsgi, what has been other solutions considered?